tests/Helper/DateTimeHelperTest.php

```diff
@@ -31,7 +31,7 @@ public function testLocale()
     {
         $localeDetector = $this->createMock('Sonata\IntlBundle\Locale\LocaleDetectorInterface');
         $localeDetector->expects($this->any())
-            ->method('getLocale')->will($this->returnValue('fr'));
```

Comment:
What was wrong with french?

Reply:
It produces a lot of errors with whitespaces for some reason

Reply:
normal char for whitespace: char(32). French char for whitespace char(194) char(160) (or something)

cs-fixer was replacing all whitespaces for the normal value, then the test were failing

Not sure how else fix this @greg0ire

Reply:
![captura de pantalla 2017-12-07 a las 22 41 05](https://user-images.githubusercontent.com/1137485/33740136-dfa9bd68-db9f-11e7-9207-0a2ce69008fa.png)

See how cs-fixer is replacing whitespaces with another whitespace?